readme.md

```diff
@@ -68,6 +68,19 @@ chalk.green(
 	chalk.blue.underline.bold('with a blue substring') +
 	' that becomes green again!'
 );
+
+// ES2015 template literals
+chalk.red(
+	`I am a red line ${calk.blue.underline.bold('with a blue substring')} ` +
```

Comment:
Indeed. I did that because the line would have been to long otherwise and if I'd continued on a second line I would have ended up with a `\n` (and spaces/tabs if I also use indentation).

I feel that the first example isn't giving you that much, maybe we should go with second only?